src/index.html

```diff
@@ -15,8 +15,73 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="stylesheet"
+      href="https://fonts.google.com/specimen/Roboto"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#">
+        <img
+          class="imagem"
+          src="./images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="links">
```

Comment:
for future BEM code (also if you wish to implement now): 
you should follow BEM structure:
links__link or something more semantic like nav__link